docs-master/Advanced/CreateUpdateTracking.md

```diff
@@ -13,7 +13,9 @@ You can add per-table support for create/update tracking. When you do this, the
 
 - When you display to the user when a thing (e.g. a Post) was modified
 
-**Note**: you _don't have to_ enable both create and update tracking. You can do either, both, or none.
+**Notes**:
+ - you _don't have to_ enable both create and update tracking. You can do either, both, or none.
+ - In your model, you cannot name the columns created_at and updated_at with a 'created_at' and 'updated_at' respectively. this causes problem with date creation and update.
```

Comment:
I read this twice, and honestly I'm really confused what you mean here

Reply:
Hello @radex !

A few days ago ... I faced a bug. The time of created_at and updated_at was always "1970-01-01T00: 00: 00.000Z". I spent hours trying to understand the problem until I decided to modify my model, especially 'created_at' and 'updated_at' I called them createdAt and updatedAt and the bug will resolved. 

I open an issue, look => https://github.com/Nozbe/WatermelonDB/issues/841 .

my English is not good. But i want help community :D

Reply:
@leo-tavares I see! I think it would be much clearer if you phrased it as something like that the defined fields must be called precisely createdAt updatedAt